### ISSUE TITLE
When returning the log file tail for a failed scoring submission, put a limit on the number of bytes returned.

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/DigitalMammographyChallenge#205]
Here is the content of the email alert that I received.
```
Dear Thomas Schaffter (tschaffter):

Your Submission to the Digital Mammography challenge (submission ID 7517857) has failed to complete its scoring phase.  The message is:

	Error encountered during prediction.  Last few lines of logs are shown below.

 Please direct any questions to the challenge forum, https://www.synapse.org/#!Synapse:syn4224222/discussion .

Sincerely,

Challenge Administration

 Logs:

STDERR: ERROR: /modelState does not exist.
```

We have decided not to return anything to the participants. For the express lane, we could allow them to retrieve all their log because we are using a dummy dataset. One of the purposes of the Express Lane is to make sure that the inference methods can read the files. However, there are multiple reasons that can make the inference method crash on the real scoring set, for example an Out of memory error. I like the idea of returning only the last few line of the log file, though I don't know what checks are performed to make sure that the participants don't use that feature to return a single log line with a lot of information.

To summarize, we can return for the scoring Express Lane the entire log file in case of errors only as long as we are using a dummy dataset. Considering the few number of times participants can score their method during the challenge, we could return an error email to the participants and invite them to contact us for additional information on why the scoring of their method failed.

**Comment by brucehoff** [comment]
> don't use that feature to return a single log line with a lot of information.

That's a good point:  We should limit the number of bytes as well as the number of lines.

> Considering the few number of times participants can score their method during the challenge

The submission limit should be a maximum number of *successful* submissions and should not include unsuccessful ones.

**Comment by tschaffter** [comment]
@brucehoff By the way, what is the limit in Bytes that you have setup? Can we close this thread?

**Comment by brucehoff** [comment]
> @brucehoff By the way, what is the limit in Bytes that you have setup?
I haven't done this yet.  Currently we return the last 25 lines.  I will add an additional constraint that those lines are less than 10KB.

**Comment by brucehoff** [comment]
https://github.com/Sage-Bionetworks/ChallengeDockerAgent/commit/a928367fa72f8ecece82554863999e3fb0f2413e
